src/main/scala/uk/gov/nationalarchives/consignmentexport/BagAdditionalFiles.scala

```diff
@@ -4,10 +4,18 @@ import java.io.File
 import java.nio.file.Path
 import cats.effect.IO
 import com.github.tototoshi.csv.CSVWriter
-import uk.gov.nationalarchives.consignmentexport.Validator.{ValidatedFFIDMetadata, ValidatedFileMetadata}
+import graphql.codegen.GetConsignmentExport.getConsignmentForExport.GetConsignment.Files
```

Comment:
Don't think this import is needed: `import graphql.codegen.GetConsignmentExport.getConsignmentForExport.GetConsignment.Files`